### DISCUSSION TITLE
Issues mentioned page && stale issues

### DISCUSSION THREAD

**Discussion by Hacksore** [community/community#65842] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Howdy 👋

Long time hub user and I have been trying to make use of the [issue mentioned](https://github.com/issues/mentioned) page to keep up with open issue. The problem is some people just stop using opensource and things you were once tagged in are no longer relevant.

Here is a simple image to explain it but this should default to something like `updated > in the last 6 months` (yes you can do this manually).

![image](https://github.com/community/community/assets/996134/4c5e9adf-38bb-439f-8cc3-02e13f1dbac4)

I do have a workaround for now which is add an `updated` label with the start of the year and that's generally good. However, it means I have to rely on a bookmark making navigating to this tab not useful by default.

**Comment by PickleNik** [comment]
> Long time hub user

I do also use GrubHub daily

**Reply by juan7732** [reply]
I do also use DockerHub daily

**Comment by Hacksore** [comment]
Tried tweeting to raise awareness for this 🤞
https://twitter.com/Hacksore/status/1709909388192825787
